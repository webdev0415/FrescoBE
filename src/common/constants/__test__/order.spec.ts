import {Order} from '../order';

describe('Order', () => {
    it('Order snapshot ', () => {
        expect(Order).toMatchSnapshot();
    });
});
