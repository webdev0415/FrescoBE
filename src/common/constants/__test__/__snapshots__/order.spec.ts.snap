// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Order Order snapshot  1`] = `
Object {
  "ASC": "ASC",
  "DESC": "DESC",
}
`;
